content/news/2021-XX-XX-keyfinder-gplv3.md

```diff
@@ -0,0 +1,30 @@
+title: "New in 2.3: KeyFinder support and license change to GPL v3"
```

Comment:
```suggestion
title: New in 2.3: KeyFinder support and license change to GPL v3
```

Reply:
Sorry, forgot to rebase on latest upstream. This is actually fixing the issue you reported (missing title). The metadata block is proper yaml now, therefore strings containing a colon need to be quoted.

Reply:
![image](https://user-images.githubusercontent.com/9455094/105648929-a193cc80-5e73-11eb-8845-96789df4d99c.png)

Reply:
oh it's fixed after the rebase